frontend/src/components/object/ObjectTable.vue

```diff
@@ -81,12 +82,20 @@ watch( getObjects, async () => {
 watch( selectedObjects, () => {
   objectStore.setSelectedObjects(selectedObjects.value);
 });
+
+// Datatable filter(s)
+const filters = ref({
+  // Need this till PrimeVue gets it together to un-break this again
+  // @ts-ignore
```

Comment:
This FilterMatchMode constant was fine up til 3.24, then it broke, then they had a "fix" that didn't fix it in 3.25, then they had another non-fix in 3.26, and that is still pending (maybe get it right in 3.27 🤷 )
https://github.com/primefaces/primevue/issues/3751#issuecomment-1482963050

Just ignore the 1 line of typescript for now

Reply:
Put in a note to revisit this again whenever we do a primevue update